host/tests/L3_TestCases/dsHost/dsHost_test1_VerifyTemperature.py

```diff
@@ -0,0 +1,160 @@
+
+        Args:
+            None.
+        """
+        super().__init__(self.testName, '1')
+
+        # Test Setup configuration file
+        self.testSetup = ConfigRead(self.testSetupPath, self.moduleName)
+
+        # Open Session for player
+        self.player_session = self.dut.getConsoleSession("ssh_player")
+
+        # Open Session for hal test
+        self.hal_session = self.dut.getConsoleSession("ssh_hal_test")
+
+        player = self.cpe.get("test").get("player")
+
+        # Get path to device profile file
+        self.deviceProfile = dir_path + "/" + self.cpe.get("test").get("profile")
+
+    def testDownloadAssets(self):
+        """
+        Downloads the artifacts and streams listed in test-setup configuration file to the dut.
+
+        Args:
+            None.
+        """
+
+        # List of streams with path
+        self.deviceDownloadPath = self.cpe.get("target_directory")
+
+        #download test artifacts to device
+        url = self.testSetup.assets.device.test1_VerifyTemperature.artifacts
+        if url is not None:
+            self.downloadToDevice(url, self.deviceDownloadPath, self.rackDevice)
+
+    def testCleanAssets(self):
+        """
+        Removes the assets copied to the dut.
+
+        Args:
+            None.
+        """
+        self.deleteFromDevice(self.testStreams)
+
+    def testRunPrerequisites(self):
+        """
+        Runs Prerequisite commands listed in test-setup configuration file on the dut.
+
+        Args:
+            None.
+        """
+
+        #Run test specific commands
+        cmds = self.testSetup.assets.device.test1_VerifyTemperature.execute
+        if cmds is not None:
+            for cmd in cmds:
+                self.writeCommands(cmd)
+
+    def testFunction(self):
+        """This function will test the get cpu tempature functionality
+
+        Returns:
+            bool
+        """
+
+        # Download the assets listed in test setup configuration file
+        self.testDownloadAssets()
+
+        # Run Prerequisites listed in the test setup configuration file
+        self.testRunPrerequisites()
+
+        # Create the dsHost class
+        self.testdsHost = dsHostClass(self.deviceProfile, self.hal_session)
+
+        self.log.stepStart(f'dsHost_test1_VerifyTemperature')
+
+        # Initialize the dsHost module
+        self.testdsHost.initialise()
+
+        # Get the CPU temperature
+        temp1 = self.testdsHost.getCPUTemperature()
+        self.log.stepStart(f'Temperature 1: {temp1}')
+
+
+        # Wait for 60 seconds
+        time.sleep(60)
+
+        # Get the CPU temperature again
+        temp2 = self.testdsHost.getCPUTemperature()
+        self.log.stepStart(f'Temperature 1: {temp2}')
+
+        # Check if the temperature is within expected range while in same mode
+        # Are we fine with this range? What validation do we want?
+        self.log.stepStart(f'Check if the temperature is within expected range')
```

Comment:
Can we ask the user to blow some hot air or put the DUT in a heat chamber and check the temp after 5 min.. to validate and confirm that there is a change in temp?

Reply:
Updated to ask them to put it into a heat chamber, or something specific to increase the temperature. As well as updated the result condition.